tds/conn.go

```diff
@@ -91,18 +91,18 @@ func NewConn(ctx context.Context, info *Info) (*Conn, error) {
 
 				caCert, err := x509.ParseCertificate(block.Bytes)
 				if err != nil {
-					return nil, fmt.Errorf("error parsing CA PEM at ssl-ca path '%s': %w",
+					return nil, fmt.Errorf("error parsing CA PEM at ssl-ca path '%s': %v",
```

Comment:
```suggestion
					return nil, fmt.Errorf("error parsing CA PEM at ssl-ca path '%s': %w",
```

`fmt.Errorf` can handle `%w`. It's just `t.<LogType>` which struggles with the format.